src/components/SubscriptionIcon/SubscriptionIcon.js

```diff
@@ -11,7 +11,7 @@ const SubscriptionIcon = ({ period, showLabel, className, isPaused }) => {
   return (
     <>
       <WrapperStyled
-        color={color || null}
+        $color={color || null}
```

Comment:
why is ' $' needed here?

Reply:
According to our a11y review: _some props passed to styled-components that shouldn't be visible in the DOM but they get rendered probably for legacy reasons. While they're not a huge issue, these legacy attributes do pollute the DOM, which in some cases might trigger issues with screen readers._

So to prevent the `color` prop from being visible in the DOM, but still passed to the element - the[ transient prop'](https://styled-components.com/docs/api#transient-props)s been applied - hence the '$'